.gitlab-ci.yml

```diff
@@ -21,14 +23,40 @@ setup:
     - bundle version
     - rake --version
 
-unit-tests:
+gcp-setup:
+  stage: setup
+  tags:
+  - gcp
+  script:
+    - docker version
+    - docker-compose version
+    - docker pull gpii/exekube:0.3.1-google
```

Comment:
if #81 is merged first, perhaps this should be updated.

Reply:
It is merged, and I updated to 0.4.0.